remoting/subsystem/src/main/java/org/jboss/as/remoting/RemoteOutboundConnectionResourceDefinition.java

```diff
@@ -72,8 +72,7 @@
             .build();
 
     public static final SimpleAttributeDefinition PROTOCOL = new SimpleAttributeDefinitionBuilder(
-            CommonAttributes.PROTOCOL, ModelType.STRING, true).setValidator(
-                    new EnumValidator<Protocol>(Protocol.class, true, false))
+            CommonAttributes.PROTOCOL, ModelType.STRING, true).setValidator(EnumValidator.create(Protocol.class))
             .setDefaultValue(new ModelNode(Protocol.HTTP_REMOTING.toString()))
             .setAllowExpression(true)
```

Comment:
same remark

Reply:
Hi, I think the `setAllowExpression(true)` is correct because the `validateParameter()` method in the `EnumValidator` class doesn't seem to check whether the value is true or false.

https://github.com/wildfly/wildfly-core/blob/main/controller/src/main/java/org/jboss/as/controller/operations/validation/EnumValidator.java#L220